addons/hw_drivers/main.py

```diff
@@ -22,7 +23,7 @@
 drivers = []
 interfaces = {}
 iot_devices = {}
-
+iot_channel = ""
```

Comment:
For iot_channel var don't put it in global.
Put this in the Manager class and make a function to get him
Odoo don't allow you to use global var